farmdata2_modules/fd2_tabs/fd2_barn_kit/seedingReport.spec.js

```diff
@@ -526,13 +530,13 @@ describe('Testing for the seeding report page', () => {
             .should('have.text', '8803')
 
             cy.get('[data-cy=direct-total-hours]')
-            .should('have.text', '7')
+            .should('have.text', '2.1')
 
             cy.get('[data-cy=direct-total-rowft-hour]')
-            .should('have.text', '3739.57')
+            .should('have.text', '4191.9')
 
             cy.get('[data-cy=direct-total-bedfr-hour]')
```

Comment:
Because I fixed the commented-out portion earlier, the values on the summary table changed (now it is correct). I had to modify the test to make it pass with the correct value (the test was incorrect before).